composer.json

```diff
@@ -13,6 +13,7 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-json": "*",
```

Comment:
Could we use a more specific version here?

Reply:
PHP extensions have no version, either they're enabled or not :)